src/contexts/auth/EthereumProviders.tsx

```diff
@@ -0,0 +1,94 @@
+import '@rainbow-me/rainbowkit/styles.css';
+
+import { getDefaultWallets, RainbowKitProvider, lightTheme } from '@rainbow-me/rainbowkit';
+import { defaultChains, configureChains, createClient, WagmiConfig } from 'wagmi';
+import { infuraProvider } from 'wagmi/providers/infura';
+import { publicProvider } from 'wagmi/providers/public';
+import merge from 'lodash.merge';
+
+// console.log('current theme values:', lightTheme());
+
+// Unfortunately, RainbowKit has no way to customize font weight
+// https://github.com/rainbow-me/rainbowkit/discussions/361
+
+// And we can't separate the selected wallet button color from
+// the "Get" button text color - they both lean on `accentColor`.
+
+// TODO: figure out if we need to customize any other options
+
+const myCustomTheme = merge(lightTheme(), {
+  colors: {
+    //   accentColor: '...',
```

Comment:
let's make this `hyperblue`, we have it in our color constants file

Reply:
both hyperblue and activeblue kinda muddy the rainbow wallet icon, lemme know if that's good with you and I'll make the change:

hyperblue:
<img width="768" alt="image" src="https://user-images.githubusercontent.com/508855/184266208-553604ce-9130-431b-aaf8-3bbfc8b91694.png">

activeblue:
<img width="761" alt="image" src="https://user-images.githubusercontent.com/508855/184266244-1a748482-e5f2-44c3-bc5e-7e61a6f585f0.png">